documentation/ci-fix.md

```diff
@@ -49,18 +49,30 @@ Or you can run it in [OpenAPI Hub](https://portal.azure-devex-tools.com/tools/st
 Refer to [Semantic and Model Violations Reference](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) for detailed description of validations and how-to-fix guidance.
 
 ## Breaking Change Check
-
+There are two kind of breaking change checks: same version breaking change check and cross version breaking change check. Both are using the 'oad' tool to compare the difference between two swaggers.
+- The same version breaking change check is for checking the existing swagger, it compares the changed swaggers to the swaggers before the change and ensure the change don't break the current version.
+- The cross version breaking change check is to find a last version swagger as a base swagger , compare the base swagger and the current swagger. Make sure the change don't break the following older versions:
```

Comment:
The cross version breaking change check usually applies to new API version scenario. It uses swaggers of the latest public version as the baseline, comparing swaggers of new version with the baseline, so that to identify if there are any updates lead to breaking experience cross versions

Reply:
The latest public version can be: